rotkehlchen/assets/converters.py

```diff
@@ -207,7 +207,7 @@ def asset_from_coinbase(cb_name: str, time: Timestamp | None = None) -> AssetWit
 
     return symbol_to_asset_or_token(GlobalDBHandler.get_assetid_from_exchange_name(
         exchange=Location.COINBASE,
-        symbol=cb_name,
+        symbol=cb_name.upper(),
```

Comment:
Please add a comment explaining what you said here. You can't put an `uppercase()` without explaining why. In the future nobody will remember why this is here.